airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -311,6 +313,21 @@ def write_ssh_config(output_path: str, original_config: Dict[str, Any], transfor
         with open(os.path.join(output_path, "ssh.json"), "w") as fh:
             json.dump(ssh_dict, fh)
 
+    @classmethod
+    def get_dbt_vars(cls, integration_type: DestinationType, config: Dict[str, Any]) -> Dict[str, Any]:
+        res = {}
+        if integration_type == DestinationType.REDSHIFT:
+            if "use_super_redshift_type" in config:
+                res["redshift_super_type"] = bool(config["use_super_redshift_type"])
```

Comment:
is input config validated by some json schema? Because `use_super_redshift_type` values don't check here.

Reply:
destination-redshift has spec.json with this param:

https://github.com/airbytehq/airbyte/blob/89b60c33a4dc0342dff8fe2e17209225f2f4ab37/airbyte-integrations/connectors/destination-redshift/src/main/resources/spec.json#L121

but for old config(s) without this parameter I still need to check:

`if "use_super_redshift_type" in config:`